server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfiles.java

```diff
@@ -101,20 +101,20 @@ public QProfile findProfileByProjectAndLanguage(long projectId, String language)
     return projectLookup.findProfileByProjectAndLanguage(projectId, language);
   }
 
-  public void addProject(int profileId, long projectId) {
-    projectOperations.addProject(profileId, projectId, UserSession.get());
+  public void addProject(String profileKey, String projectUuid) {
```

Comment:
Moreover, QProfiles is deprecated (The class to use should be QProfileService, but should now be removed)